client/src/components/MultiJurisdictionAudit/Setup/Review/index.tsx

```diff
@@ -136,139 +121,163 @@ const Review: React.FC<IProps> = ({ prevStage, locked, refresh }: IProps) => {
-        </thead>
-        <tbody>
-          {targetedContests.map((c: IContest) => (
-            <tr key={c.id}>
-              <td>{c.name}</td>
-              <td>{c.jurisdictionIds.join(', ')}</td>
+      <ElevatedCard>
+        <SettingsTable>
+          <tbody>
+            <tr>
+              <td>Election Name:</td>
+              <td>{electionName}</td>
             </tr>
-          ))}
-        </tbody>
-      </ContestsTable>
-      <ContestsTable>
-        <thead>
-          <tr>
-            <th>Opportunistic Contests</th>
-            <th>Jurisdictions</th>
-          </tr>
-        </thead>
-        <tbody>
-          {opportunisticContests.map((c: IContest) => (
-            <tr key={c.id}>
-              <td>{c.name}</td>
-              <td>{c.jurisdictionIds.join(', ')}</td>
+            <tr>
+              <td>Risk Limit:</td>
+              <td>{riskLimit}</td>
             </tr>
-          ))}
-        </tbody>
-      </ContestsTable>
-      <H4>Sample Size Options</H4>
+            <tr>
+              <td>Random Seed:</td>
+              <td>{randomSeed}</td>
+            </tr>
+            <tr>
+              <td>Participating Jurisdictions:</td>
+              <td>
+                <a
+                  href={`/api/election/${electionId}/jurisdiction/file/csv`}
+                  target="_blank"
+                  rel="noopener noreferrer"
+                >
+                  {jurisdictionFile && jurisdictionFile.file
+                    ? jurisdictionFile.file.name
+                    : ''}
+                </a>
+              </td>
+            </tr>
+            <tr>
+              <td>Audit Board Data Entry:</td>
+              <td>{online ? 'Online' : 'Offline'}</td>
+            </tr>
+          </tbody>
+        </SettingsTable>
+        <ContestsTable>
+          <thead>
+            <tr>
+              <th>Target Contests</th>
+              <th>Jurisdictions</th>
+            </tr>
+          </thead>
+          <tbody>
+            {targetedContests.map((c: IContest) => (
+              <tr key={c.id}>
+                <td>{c.name}</td>
+                <td>{c.jurisdictionIds.join(', ')}</td>
+              </tr>
+            ))}
+          </tbody>
+        </ContestsTable>
+        <ContestsTable>
+          <thead>
+            <tr>
+              <th>Opportunistic Contests</th>
+              <th>Jurisdictions</th>
+            </tr>
+          </thead>
+          <tbody>
+            {opportunisticContests.map((c: IContest) => (
+              <tr key={c.id}>
+                <td>{c.name}</td>
+                <td>{c.jurisdictionIds.join(', ')}</td>
+              </tr>
+            ))}
+          </tbody>
+        </ContestsTable>
+      </ElevatedCard>
+      <br />
+      <H4>Estimated Sample Sizes</H4>
       <Formik
         initialValues={{
-          sampleSize: sampleSizeOptions.length ? sampleSizeOptions[0].size : '',
-          customSampleSize: '',
+          sampleSizes: initialValues,
```

Comment:
does this need to be nested, or could it just be `intialValues={initialValues}` now?

Reply:
@jonahkagan Formik needs a root object to name things, and if we made that the undeterministic contest ids, that would make things more confusing for referencing within the code, so I think this is the best way to do it.